docs/conf.py

```diff
@@ -133,7 +133,7 @@
 # built documents.
 #
 # The full version, including alpha/beta/rc tags.
-release = "0.2.1"
+release = "0.2.2-dev"
```

Comment:
Does it make sense to set
```python
release = jet.__version__
```
here, so that it doesn't always need to be manually updated?

Reply:
Excellent suggestion! The idea of manually specifying the version here precedes the Jet Python package.